cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,77 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import { settingsPage } from '../support/pages/settings_pageObject';
+import { homePageObject } from '../support/pages/home.pageObject';
+import { profilePage } from '../support/pages/profile_pageObject';
+
 describe('Settings page', () => {
+  let user;
+  let nextUser;
   before(() => {
```

Comment:
In before, you can combine both calls of cy.task into one promise to avoid nested then

`before(() => {
  cy.task('generateUser').then((generateUser) => {
    user = generateUser;
    return cy.task('generateUpdateData');
  }).then((generateUpdateData) => {
    nextUser = generateUpdateData;
  });
});
`

Reply:
I've changed this. ty for the comment, it starts working better!